features/admin.core.v1/configs/app.ts

```diff
@@ -121,6 +121,7 @@ export class Config {
             appHomePath: window[ "AppUtils" ]?.getConfig()?.routes?.home,
             appLoginPath: window[ "AppUtils" ]?.getConfig()?.routes?.login,
             appLogoutPath: window[ "AppUtils" ]?.getConfig()?.routes?.logout,
+            centralDeploymentEnabled: window[ "AppUtils" ]?.getConfig()?.routes?.centralDeploymentEnabled,
```

Comment:
In getconfig function in console/src/init/app-utils, this value is returned in the config object as a first level property, not as a nested property of routes object.

```suggestion
            centralDeploymentEnabled: window[ "AppUtils" ]?.getConfig()?.centralDeploymentEnabled,
```